src/bin/lfortran.cpp

```diff
@@ -1227,6 +1234,13 @@ int link_executable(const std::vector<std::string> &infiles,
                 std::cout << "The command '" + cmd + "' failed." << std::endl;
                 return 10;
             }
+            if (outfile == "a.out") {
+                err = system("./a.out");
+                if (err) {
+                    std::cout << "The command './a.out' failed." << std::endl;
+                    return 10;
+                }
+            }
```

Comment:
```suggestion
            if (outfile == "a.out") {
                system("./a.out");
            }
```
I think this would work.